.env.example

```diff
@@ -1,4 +0,0 @@
-MNEMONIC=''
```

Comment:
in the previous PR I commented on the .env file, while it's fine to keep the .env.example in order to show how to configure the repo